sympy/printing/mathematica.py

```diff
@@ -109,6 +109,9 @@ def _print_Integral(self, expr):
     def _print_Sum(self, expr):
         return "Hold[Sum[" + ', '.join(self.doprint(a) for a in expr.args) + "]]"
 
+    def _print_Derivative(self, expr):
+        return "D[" + ', '.join(self.doprint(a) for a in expr.args) + "]"
```

Comment:
Shouldn't you add a `Holf[` before `D`? Derivative objects is SymPy mean unevaluated derivatives.

Look at line 110, just here above. It is `Holf[Sum[ ... ...`.